src/50_Visual_Effects/Basics_of_orientation_effects.html

```diff
@@ -0,0 +1,304 @@
+
+      Praesent non arcu volutpat, maximus risus in, rutrum tellus. Quisque nec porttitor purus. Quisque hendrerit erat consequat dignissim vehicula. Quisque tortor orci, bibendum et lacinia at, porttitor nec est. Nunc venenatis sollicitudin dui, et fermentum lectus euismod vitae. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Phasellus diam ante, sagittis non finibus sed, pulvinar id dolor. Maecenas ac facilisis lacus. Suspendisse potenti.
+
+      Curabitur dignissim consequat ante, quis facilisis ipsum faucibus nec. Suspendisse potenti. Mauris lorem eros, finibus id tincidunt quis, bibendum at urna. Aenean quis tortor ultricies, sodales ipsum id, imperdiet felis. Maecenas vestibulum cursus est convallis imperdiet. In convallis tempus lacus, suscipit tempus turpis mattis id. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Proin massa massa, molestie eu pretium in, eleifend vel erat. Pellentesque dictum turpis sit amet neque elementum aliquam a sed metus. Donec dignissim laoreet massa vel vehicula. Praesent pretium arcu consequat augue dignissim, at ultrices quam placerat. Proin justo odio, fermentum sed malesuada eu, facilisis non quam. In molestie enim nec ligula tempus, sed condimentum dui molestie. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec lobortis, lacus cursus varius malesuada, lacus enim blandit ipsum, non hendrerit est sapien et ex. Nullam quis pulvinar nulla, vitae venenatis massa.
+
+      In mattis at eros id ullamcorper. Aenean ac hendrerit magna. Sed volutpat sapien quis justo accumsan eleifend. Praesent lobortis, velit sed elementum molestie, urna nibh malesuada lacus, vitae efficitur tellus elit ac augue. Aliquam erat volutpat. Praesent ultricies felis erat, ut luctus nulla pellentesque in. Etiam lacinia neque id neque egestas tempor.      
+      <h3>End of text</h3>
+    </div>
+  </main>
+
+  <!--
+  The animation here is a translation along the Y axis to mimic the scrolling action.
+  -->
+
+  <amp-animation id="anim" layout="nodisplay">
+    <script type="application/json">
+      {
+      "duration": "3s",
+      "fill": "both",
+      "direction": "alternate",
+      "animations": [
+        {
+          "selector": "#ipsum",
+          "keyframes": [
+            { "transform": "translateY(0)" },
+            { "transform": "translateY(-1170px)" }
+          ]
+        }
+      ]
+    }
+    </script>
+  </amp-animation>
+
+  <!--
+  ## Tilt based translation animations
+
+  Let's create a tilt based animation where the clock's hand moves 
+  as you tilt the device along a specific axis.
+  -->
+
+  <!--
+  The first animation is along the `beta` axis, to see this in action rotate your phone along the y axis. 
+
+  Here you can see that we are using the `beta` event to progress through the animation.
```

Comment:
Can you merge the three last sentences into a single paragraph? I think looks a bit weird that every sentence is a separate paragraph.

Reply:
Done.